Civi/Test/Api4TestTrait.php

```diff
@@ -0,0 +1,306 @@
+<?php
+
+namespace Civi\Test;
+
+use Civi\Api4\Generic\Result;
+use Civi\Api4\Utils\CoreUtil;
+
+/**
+ * Class Api3TestTrait
+ *
+ * @package Civi\Test
+ *
+ * This trait defines a number of helper functions for testing APIv4.
```

Comment:
I'd find it easier to understand with some kind of pointer about how to use the class. (*Which of the 8 helpers should I care about?*) Maybe like:

```suggestion
 * Define helpers for generating, tracking, and cleaning test-data with APIv4. Specifically:
 * - `createTestRecord()` will autofill a new record with example data.
 * - `deleteTestRecords()` will batch-delete all the test-data created by `createTestRecord()`
```

Reply:
Yep - I think those might be the only ones we would encourage people to use so that's probably right - I'm trying to privatise as much else as I can

Reply:
@totten I've updated this but what I found is there are 2 functions

`createTestRecord()`
`saveTestRecords()`

I suppose that's consistent with apiv4 but it feels like they should use the same verb - given it would be work to change I am VERY happy to be convinced otherwise

Reply:
Yea, they both use the APIv4 `save` action, so changing to `saveTestRecord()` would be more consistent all-round.